### DISCUSSION TITLE
point in streamlines

### DISCUSSION THREAD

**Discussion by cxmDREAM** [dipy/dipy#3327] [Reconstruction]
Hello, I am a bit confused about using streamlines for post-processing, and I hope you can help me clarify it
_https://docs.dipy.org/stable/examples_built/visualization/viz_roi_contour.html#_
**`streamlines** = Streamlines(streamlines)`
and
_https://docs.dipy.org/stable/examples_built/fiber_tracking/linear_fascicle_evaluation.html_
```
candidate_sl_sft = load_trk(lr_superiorfrontal_path, 'same')
candidate_sl_sft.to_vox()
**candidate_sl** = candidate_sl_sft.streamlines
```
What is the difference between streamlines and candidate_stl? Which coordinate should I use to draw streamlines using the (x, y, z) coordinate points? Or how to obtain it?

**Comment by pjsjongsung** [comment]
Hello! streamlines above are ones that were calculated, and the ones below are ones that were loaded from a file. So it is just a different situation of how to obtain the streamlines. I am guessing you are wondering what visualization method to use?
